doc/releases/changelog-dev.md

```diff
@@ -4,7 +4,48 @@
 
 <h3>New features since last release</h3>
 
-* The text based drawer accessed via `qml.draw` has been overhauled.
+* Transform a circuit from quantum tape, quantum function or quantum node to a pairwise
+  commutation DAG (directed acyclic graph). The node represents the quantum operations, and the edges represent 
+  non commutation.
+  [(#1712)](https://github.com/PennyLaneAI/pennylane/pull/1712)
+  
+  From the following quantum function,
+  ```
+  def circuit(x, y, z):
+      qml.RX(x, wires=0)
+      qml.RX(y, wires=0)
+      qml.CNOT(wires=[1, 2])
+      qml.RY(y, wires=1)
+      qml.Hadamard(wires=2)
+      qml.CRZ(z, wires=[2, 0])
+      qml.RY(-y, wires=1)
+      return qml.expval(qml.PauliZ(0))
+  ```
+  the commutation DAG can be returned by using the following code:
+  ```
+  get_dag = commutation_dag(circuit)
+  theta = np.pi/4
+  phi = np.pi/3
+  psi = np.pi/2
+  dag = get_dag(theta, phi, psi)
+  ```
+  You can access all nodes by using the get_nodes function in the form of a list (ID, CommutationDAGNode):
+  ```
+  nodes = dag.get_nodes()
+  ```
+  You can also access specific nodes CommutationDAGNode by using get_node function. From the CommutationDAGNode
+  you can directly access all node attributes.
+  ```
+  second_node = dag.get_node(2)
+  second_operation = second_node.op
+  second_node_successors = second_node.successors
+  second_node_predecessors = second_node.predecessors
+  ```
```

Comment:
@rmoyard don't forget to update the code example here to:

1. use pycon syntax!
2. Show the output, like you do in the `CommutationDAG` docstring :)

Reply:
Thanks @josh146 !